e2e/cypress/support/api/system.js

```diff
@@ -152,3 +152,16 @@ function uploadLicenseIfNotExist() {
         });
     });
 }
+
+Cypress.Commands.add('apiDeleteBrandImage', () => {
+    return cy.request({
+        headers: {'X-Requested-With': 'XMLHttpRequest'},
+        url: '/api/v4/brand/image',
+        method: 'DELETE',
+        failOnStatusCode: false,
+    }).then((response) => {
+        // both deleted and not existing responses are valid
+        expect(response.status).to.be.oneOf([200, 404]);
+        return cy.wrap({response});
```

Comment:
For consistency if not returning specific object.
```suggestion
        return cy.wrap(response);
```